torch/nn/init.py

```diff
@@ -80,12 +80,18 @@ def calculate_gain(nonlinearity, param=None):
     SELU              :math:`\frac{3}{4}`
     ================= ====================================================
 
+    .. warning::
+        In order to implement `Self-Normalizing Neural Networks`_ ,
+        you should use `nonlinearity='linear'` instead of `nonlinearity='selu'`.
```

Comment:
Can you expand on this slightly to explain the reason why `nonlinearity='linear'` is preferred (i.e. because that sets the gain to 1 and that's what the paper uses)?

Also, formatting nit: use double backticks instead of single around the code snippets